### PR TITLE
feat: [CET-1197]: fix OverviewProps type for reuse in ET MFE

### PR DESCRIPTION
**Pull request by guycastel**
### Summary
Small fix for `OverviewProps` type/interface which is used in ET MFE.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by guycastel** [comment]
retrigger sonar

File: src/modules/70-pipeline/components/Notifications/Steps/Overview.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { useStrings } from 'framework/strings'
 import type { NotificationRules } from 'services/pipeline-ng'
 import { NameSchema } from '@common/utils/Validation'
 
-export interface OverviewProps {
+export type OverviewProps = StepProps<NotificationRules> & {
```
Comment (abhinavrastogi-harness) [inline src/modules/70-pipeline/components/Notifications/Steps/Overview.tsx:17]:
```suggestion
export interface OverviewProps extends StepProps<NotificationRules> {
```

**Comment by guycastel** [comment]
retrigger cypress-cd

**Review by knagurski — APPROVED** [review]
Makes sense 👍🏻

**Review by Sagar-Rajput7 — APPROVED** [review]

**Pull request merged**